src/webview/MessageListWeb.js

```diff
@@ -12,18 +12,42 @@ import { getInputMessages } from './webViewHandleUpdates';
 import * as webViewEventHandlers from './webViewEventHandlers';
 import { base64Utf8Encode } from '../utils/encoding';
 
-export default class MessageListWeb extends Component<Props> {
+type State = {
+  isWebViewReady: boolean,
+};
+
+export default class MessageListWeb extends Component<Props, State> {
   context: Context;
   props: Props;
   webview: ?Object;
-  isReady: boolean;
   unsentMessages: WebviewInputMessage[] = [];
 
+  state = {
+    isWebViewReady: false,
+  };
+
   static contextTypes = {
     styles: () => null,
     theme: () => null,
   };
 
+  /*
+  *
+  * Synchronizing setup with webview
+  *
+  */
+  sendTestPing = (): void => {
+    if (this.state.isWebViewReady) {
+      return;
+    }
+    this.sendMessages([{ type: 'ping' }]);
+    setTimeout(this.sendTestPing, 10);
```

Comment:
I am quite concerned about this value. Sending messages to the WebView is not slow, but it isn't just a normal a method call either. Everything goes through the bridge (somewhat slow) and then is executed as simple `MessageEvent` (maybe fast enough)

Reply:
So should we increase or decrease this value?
```suggestion
    setTimeout(this.sendTestPing, 5);
```

or

```suggestion
    setTimeout(this.sendTestPing, 15);
```

Reply:
I was thinking somethinkg like 100 ;)